client/js/table-view.js

```diff
@@ -78,15 +80,43 @@ class TableView {
 		this.sheetBodyEl.appendChild(fragment);
 	}
 
+	renderTableFooter() {
```

Comment:
Looks like the files are still set to convert to tabs. Please see the #general channel for how to convert your current files to "Tab Width: 2 spaces" and "Convert indentation to spaces". Also change your Sublime Text preferences to do both things.